src/components/AudioTrack/PlayPause.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <button
     type="button"
-    class="flex items-center justify-center bg-dark-charcoal h-20 w-20 rounded-sm transition-shadow duration-100 ease-linear disabled:opacity-70 focus:outline-none focus-visible:ring focus-visible:ring-offset-2 focus-visible:ring-pink"
-    @click="toggle"
+    class="play-pause flex items-center justify-center bg-dark-charcoal text-white transition-shadow duration-100 ease-linear disabled:opacity-70 focus:outline-none focus-visible:ring focus-visible:ring-offset-2 focus-visible:ring-pink"
```

Comment:
Where does the `play-pause` class come from?

Reply:
It's not a utility that's coming from any library. The `play-pause` class is declared here to allow the layouts to style the component from their stylesheets by referencing it.

Reply:
Ah okay... In generally I prefer to avoid that approach because it creates the problem of "magic strings" that are hard to change or understand in the future. Would it be possible to in the future instead pass a new prop for the specific styles that should be added to the component? For example allow the user to pass a `button-class` prop or something like that would allow them to define specific scoped styles or use CSS Modules for example and would be easier tie the heirarchy of style application (at least in my experience).